a3p-integration/package.json

```diff
@@ -4,12 +4,14 @@
         "fromTag": "main"
     },
     "scripts": {
-        "build": "echo Use synthetic-chain to build proposal images",
-        "test": "echo Use synthetic-chain to test proposal images"
+        "build": "./node_modules/.bin/synthetic-chain build",
+        "test": "./node_modules/.bin/synthetic-chain test",
+        "doctor": "./node_modules/.bin/synthetic-chain doctor"
```

Comment:
shouldn't plain `"synthetic-chain build"` work? doesn't `yarn` arrange for `./node_modules.bin` to be in `$PATH`?

Reply:
It should but it doesn't due to the `.ts` extensions. Another option is to use `tsx` as in https://github.com/Agoric/agoric-3-proposals/pull/79/commits/e4c479125805a2cd15cf2c60afc5173476ccb6f9